CodingDojo/games/expansion/src/main/java/com/codenjoy/dojo/expansion/services/Scores.java

```diff
@@ -92,8 +101,29 @@ public void update(Object data) {
     @Override
     public JSONObject getScore() {
         scores.put(SCORE, score);
+        scores.put(DETAILS, details());
         return scores;
     }
+
+    private String details() {
+        List<Object> list = rounds().toList();
+        int count = list.size();
+        int sum = list.stream()
+                .map(it -> (Integer) it)
```

Comment:
this line is useless
you can map to int immediately
`list.stream().mapToInt(obj -> (Integer) obj).sum()`

Reply:
Done